kernel/kernel-config-4.14.x

```diff
@@ -0,0 +1,2740 @@
+#
```

Comment:
we already have a configuration file [1], could you modify these scripts to use it?

[1] - https://github.com/kata-containers/packaging/tree/master/kernel/configs